tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowserlocationchange_event.html

```diff
@@ -55,5 +55,16 @@
 
 });
 
+async_test(function(t) {
+  var iframe = document.createElement("iframe");
+  iframe.mozbrowser = "true";
+  iframe.src = "mozbrowserlocationchange_event_iframe.html";
+  iframe.addEventListener("mozbrowserlocationchange", function(e) {
```

Comment:
Don't know if this would help, there aren't any asserts in the code, just a call to `t.done()` on termination. The reason for this is that we actually get two `mozbrowserlocationchange` events, one for the initial load and one for the location change. We have to ignore the first one.

@paulrouget actually, is this showing up a bug? Are we meant to be generating `mozbrowserlocationchange` events just on a location change, not on the initial location?